PureBasicIDE/AutoComplete.pb

```diff
@@ -29,6 +29,12 @@ Global AutoComplete_CurrentModule$
 Global NewMap AutoComplete_LastStructureItem.s()  ; key and value is lowercase
 Global NewMap AutoComplete_LastModuleItem.s()
 
+;- Added by mk-soft; Date 29.05.2020; Linux force redraw TreeView
```

Comment:
This comment is no longer necessary because Git stores this information about who added, changed, and removed which line.